spec/libtest/test_subject.h

```diff
@@ -51,6 +51,26 @@ typedef struct _TestStruct {
  * TestSubject:
  *
  * Main class used to test all sort of things directly related or not to GObject.
+ * 
+ * Example docs for testing:
+ *
+ * getter: [method@Test.Subject.get_crystallize_docs]
+ * 
+ * setter: [method@Test.Subject.set_crystallize_docs]
```

Comment:
You can use `test_subject_set_str_list` to test this, to avoid create an empty function.